index.d.ts

```diff
@@ -1,6 +1,7 @@
 // Type definitions for webstomp-client v1.0.x
 // Project: https://github.com/JSteunou/webstomp-client
 // Definitions by: Jimi Charalampidis <https://github.com/JimiC>
+//                 Rodolfo Aguirre <https://github.com/roddolf>
```

Comment:
The correct way is to add a comma separator between the contributors.

Reply:
Ups, true.